docs/dkfz.md

```diff
@@ -0,0 +1,19 @@
+## DKFZ configuration
+
+This is a configuration file has been used in DKFZ HPC cluster. 
+
+To use, run the pipeline with -profile dkfz. This will download and launch the dkfz.config which has been pre-configured with a setup suitable for the automatically created cluster. Using this profile, a Docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
```

Comment:
Is that cluster really created automatically for you or is it just available?

Reply:
it is available.

Reply:
Then I would reformulate the above - it sounds like its dynamically created once you run a job which is not the case here ;-)